extension/docs/HowToRun.md

```diff
@@ -102,7 +100,8 @@ Refer to our [docker hub](https://hub.docker.com/r/commercetools/commercetools-a
     commercetools/commercetools-adyen-integration-extension
 ```
 
-### AWS Lambda
+## Deployment
 
-1. For deployment to AWS Lambda, zip the extensions folder and specify `src/lambda.handler` as the entry point for the AWS Lambda function.
-2. When the extension module is run as AWS Lambda **it will NOT create** the required resources like custom types or commercetools API extension for you. Please follow the [manual resource creation guide](#creating-resources-manually) instead. 
+Extension module supports different deployment [options](/deployment-examples). 
```

Comment:
Could you link the GCP readme instead of linking to the deployment-examples directory.

Reply:
but that will express the idea that we only support GCP deployment which is not true.

Reply:
Then don't link the AWS lambda in here, the user can navigate, which does not make sense to me.

Reply:
1ccf903e31fc2bb38b11f2dca34fc88d09da8879